api/src/main/java/org/pine/api/Pine.java

```diff
@@ -28,6 +30,25 @@ public static void init(String mode, String address) {
 
   }
 
+  @SuppressWarnings("unchecked")
+  public static void init(String mode, String address,int index) {
```

Comment:
I think we should add a new `init` for this. It's better to move this code to the `init` at line8.